internal/lock/lockfile.go

```diff
@@ -127,6 +128,10 @@ func IsLegacyPackage(pkg string) bool {
 		!strings.HasPrefix(pkg, "/")
 }
 
+func (l *File) Tidy(project devboxProject) {
+	l.PackageMap = lo.PickByKeys(l.PackageMap, project.Packages())
```

Comment:
This doesn't need a parameter. The lockFile struct has access to devbox struct (i.e. `l.devboxProject.Packages()`)

Reply:
oh good point